leaf-ui/js/onFunctions.js

```diff
@@ -236,7 +236,16 @@ graph.on("add", function (cell) {
         createdInstance = createdInstance.filter(view => view.model instanceof joint.shapes.basic.Actor);
 
         // Create placeholder name based on the number of instances
-        var name = cell.attr('.name/text') + "_" + (createdInstance.length - 1);
+        if (createdInstance.length >= 2){
+            var lastactor = createdInstance[createdInstance.length - 2].model.attr('.name/text');
+            var num = Number.parseInt(lastactor.charAt(lastactor.length-1))+1
```

Comment:
Do you think we could inline the `num`? Since it's only used once here

Reply:
Very valid, I'll change it